packages/lesswrong/components/comments/CommentsItem/CommentPollVote.tsx

```diff
@@ -28,10 +29,14 @@ const CommentPollVote = ({ comment, classes }: { comment: CommentsList; classes:
   const { document: forumEvent, loading } = useSingle({
     documentId: comment.forumEventId ?? '',
     collectionName: "ForumEvents",
-    fragmentName: 'ForumEventsMinimumInfo',
-    skip: !comment.forumEventId
+    fragmentName: 'ForumEventsDisplay',
+    skip: !comment.forumEventId,
+    ssr: false
   });
 
+  const isGlobal = forumEvent?.isGlobal !== false;
+  const pollLink = (!isGlobal && forumEvent) ? `#${forumEvent._id}` : undefined;
+
   const agreeWording = forumEvent?.pollAgreeWording || "agree";
```

Comment:
Are the defaults meant to be lowercase? I think they used to be capitalized, no?

Reply:
Nope they were always lower case 🤷‍♂️